### DISCUSSION TITLE
#52 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by 0xPabloLI** [tog-ETH-er/togETHer#35] [General]
Agenda:

-Community updates
-Using google meeting
-Website update
-Shared event calendar
-Fireside discussion

会议主题：togETHer
会议时间： 21:00-22:00
重复周期：每两周 (周一)

点击链接入会，或添加至会议列表：
https://meeting.tencent.com/dm/ScpRTdRfaT3s

会议 ID：377-8690-8773

**Comment by bambalados** [comment]
Crypto Tech Night

- CTN#19 - MPC 钱包与Account Abstraction (暂定日期：12月15号）

**Comment by 0xPabloLI** [comment]
PlanckerDAO

- 11.19 [Otterspace/EIP4973 分享](https://plancker.notion.site/14-88038c12583d4d8589d2426de40e6b6f)
- 11.26 [小V共读会4：What do I think about network states?](https://plancker.notion.site/15-V-4-eef8c2f199a946b0babca73b39cc0de8)

**Comment by 0x-stan** [comment]
# Dapp-Learning

## repo update

18个PR #872 ～ #890

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了2场分享会

- 《Tornado cash Decoding  ( ETHKL1 & Dapp-Learning  ) 》 by cheechyuan
  - https://www.youtube.com/watch?v=VtUOPUZ1qy8

- 《Iden3: 应用/原理/源码/实战》by arc
  - https://www.youtube.com/watch?v=DXGTALBUcJ4
  - https://www.youtube.com/watch?v=4ENECj3Jw24

**Comment by Riely007** [comment]
### 以太坊翻译小组

- 官网 review 进行中
- idea: local glossary

**Comment by Stamp9** [comment]
### 原语里弄
[第七届网络社会年会上海论坛](https://mp.weixin.qq.com/s/OXGmL6yfW245E7AwOaQgWQ)

**Comment by xrdavies** [comment]
### Rebase 社区
- 14篇 Web3极客日报
- [2022 Rebase 黑客松启动](https://github.com/rebase-network/2022Q4-hackathon)，筹备进行中
- Who-Is-Hiring 若干职位
